File: VB/GetValue/XtraReport1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports DevExpress.XtraReports.UI
Imports System.Data

Namespace GetValue
	Partial Public Class XtraReport1
		Inherits DevExpress.XtraReports.UI.XtraReport
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub calculatedField1_GetValue(ByVal sender As Object, ByVal e As GetValueEventArgs) Handles calculatedField1.GetValue
			Dim view As DataRowView = TryCast(e.Row, DataRowView)
			e.Value = GetCurrentColumnValue("CategoryName") & " " & view.Row("UnitPrice")
		End Sub

	End Class
End Namespace

File: CS/GetValue/XtraReport1.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using DevExpress.XtraReports.UI;
using System.Data;

namespace GetValue
{
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport
    {
        public XtraReport1()
        {
            InitializeComponent();
        }

        private void calculatedField1_GetValue(object sender, GetValueEventArgs e)
        {
            DataRowView view = e.Row as DataRowView;
            e.Value = GetCurrentColumnValue("CategoryName") + " " + view.Row["UnitPrice"];
        }

    }
}
